docs/DeveloperGuide.md

```diff
@@ -467,22 +467,22 @@ schedules, and provides them with a platform to visualise defensive and offensiv
+|----------|---------------------------------|-----------------------------------------------------------------|------------------------------------------------------------------------------|
+| `* * *`  | forgetful coach                 | enter team-specific or player note                              | look up these information                                                    |
+| `* * *`  | coach                           | delete team-specific or player note                             | keep these information relevant and up-to-date                               |
+| `* * *`  | forgetful coach                 | remember the names of players on my team                        | look up them in case I forget                                                |
+| `* * *`  | disorganized                    | add and tag new roles/teams to a contact                        | easily retrieve relevant information                                         |
+| `* * *`  | coach                           | easily retrieve contact information of relevant parties         | quickly broadcast information to them                                        |
+| `* * *`  | organized coach                 | view players by their strengths and weaknesses                  | make informed decision on choosing the best person                           |
+| `* * *`  | disorganized coach              | add existing and upcoming tasks                                 | keep track of my schedule easily                                             |
+| `* * *`  | disorganized coach              | delete wrongly added or past tasks                              | organize my schedule better                                                  |
+| `* * *`  | strategic coach                 | save current formation information                              | plan my game plays and save strategies                                       |
+| `* *`    | lazy and forgetful coach        | view a list of help commands and their descriptions             | easily recall how to do a specific task                                      |
+| `* *`    | organised coach                 | view players by their strengths and weaknesses                  | make informed decisions on choosing the best person for a specific objective |
+| `**`     | organized and data-driven coach | sort my players by strengths and weaknesses                     | get a better understanding of how players compare                            |
+| `*`      | strategic coach                 | change the position of players (x-y coordinate) during the game | ensure my team works together                                                |
+| `*`      | coach                           | drag and drop a player into a calendar                          | plan scheduled events for them according to their needs                      |
```

Comment:
I think this user story can be removed as we are not planning to implement a calendar in our program

Reply:
Is it necessary to delete given that it's given low priority (under the section "unlikely to have"?

Reply:
You're right. I guess we can keep this user story